src/openrct2/PlatformEnvironment.cpp

```diff
@@ -182,6 +186,7 @@ const char * PlatformEnvironment::FileNames[] =
     "hotkeys.dat",          // CONFIG_KEYBOARD
     "objects.idx",          // CACHE_OBJECTS
     "tracks.idx",           // CACHE_TRACKS
+    "RCTdeluxe_install" PATH_SEPARATOR "Data" PATH_SEPARATOR "mp.dat", // MP_DAT
```

Comment:
This looks suspicious to me. Has this actually been tested on non-Deluxe versions?